contracts/utils/run_script.sh

```diff
@@ -4,20 +4,34 @@
 source .env
 
 # Read script
-echo Which script do you want to run?
-read script
+# echo Which script do you want to run?
+# read script
+# DEPLOYER_SCRIPT=$1
 
 # Read script arguments
-echo Enter script arguments, or press enter if none:
-read -ra args
+# echo Enter script arguments, or press enter if none:
+# read -ra args
 
 # Run the script
-echo Running Script: $script...
+echo Running Script: $DEPLOYER_SCRIPT_FACTORY...
 
 # Run the script with interactive inputs
-forge script $script \
-    --rpc-url $RPC_URL \
+forge script script/$DEPLOYER_SCRIPT_FACTORY \
+    --chain celo-alfajores \
+    --rpc-url $ALFAJORES_RPC_URL \
```

Comment:
Minor nitpick: if you deploy to a different RPC url, shouldn't it have a generic name? I.e. Alfajores points to a single chain/environment, and you may want to deploy other places.

Reply:
yes this was to make it easier to deploy to Celo Alfajores from this branch but this should not be merged to keep it generic.